src/app/Welcome/Welcome.tsx

```diff
@@ -31,11 +44,15 @@ const Welcome: React.FunctionComponent<any> = props => {
     'Use it as a cache or a data grid. Works on bare-metal, containerized and virtualized environments. ' +
     'Works on AWS, Azure, Google Cloud, Kubernetes and OpenShift.';
```

Comment:
Feel we could drop this sentence. "Cloud-native environments" in the previous sentence covers these platforms.